pre-docs/writing_tasks.md

```diff
@@ -206,44 +207,35 @@ and the path to the task within the module.
 
 You can write tasks in any language that will run on the target nodes. Give
 task files the extension for the language they are written in (such as `.rb` for
-Ruby), and place them in your module's `./tasks` directory.
+Ruby), and place them in the top level of your module's `./tasks` directory.
 
 Task names are composed of two or more name segments, indicating:
```

Comment:
This sentence seems incorrect. Task names are either 1 or 2 segments. 1 for `init`